rules/REQUEST-921-PROTOCOL-ATTACK.conf

```
@@ -305,6 +305,33 @@ SecRule REQUEST_URI "@rx unix:[^|]*\|" \
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
+# Detection for old V1 cookie format from RFC 2109.
+#
+# This has been abused by the cookie sandwich technique, in diverse issues affecting Apache Tomcat, Python, and maybe others.
+# RFC 6265 deprecated and replaced RFCs 2109 and 2965.
+# It completely removed "$Version", meaning user agents and servers no longer use this attribute.
+# See:
+# - https://portswigger.net/research/stealing-httponly-cookies-with-the-cookie-sandwich-technique
+# - https://github.com/owasp-modsecurity/ModSecurity/wiki/Reference-Manual-%28v2.x%29#seccookieformat
+SecRule REQUEST_COOKIES:/\"?\$Version/ "@streq 1" \
```

```suggestion
SecRule REQUEST_COOKIES:/\x22?\x24Version/ "@streq 1" \
```
This solves the libmodsecurity3 problem (and still works on Apache2).

---

Perfect!